ship.config.js

```diff
@@ -11,7 +11,7 @@ export default {
     packagesToPublish: packages,
   },
   publishCommand({ tag }) {
-    return `yarn publish --access public --tag ${tag}`;
+    return `npm publish --tag ${tag}`;
```

Comment:
Isn't there something to put for the token to use it on the ci?

Reply:
it should automatically pick up the `NPM_AUTH_TOKEN` that's on the CI